include/MidiClip.h

```diff
@@ -63,7 +63,8 @@ class LMMS_EXPORT MidiClip : public Clip
 	// note management
 	Note * addNote( const Note & _new_note, const bool _quant_pos = true );
 
-	void removeNote( Note * _note_to_del );
+	NoteVector::const_iterator removeNote(NoteVector::const_iterator note);
```

Comment:
Should this be `public`? Also, the name `note` doesn't look like an iterator, too.

Reply:
> Should this be `public`?

Yes?

> Also, the name `note` doesn't look like an iterator, too.

What should we name it then?

Reply:
> > Should this be `public`?
> 
> Yes?

I mean, is this function intended to be used internally, or available publicly?

> What should we name it then?

I think `it` or `pos`(like STL functions) would be better.

Reply:
> I mean, is this function intended to be used internally, or available publicly?

This is used from the Piano Roll, so public as far as I can tell.

> I think `it` or `pos`(like STL functions) would be better.

Can do. 👍